spartan/aztec-network/eth-devnet/run-locally.sh

```diff
@@ -3,4 +3,4 @@
 REPO_ROOT=$(git rev-parse --show-toplevel)
 
 ${REPO_ROOT}/spartan/aztec-network/eth-devnet/create.sh
-(cd ${REPO_ROOT}/spartan/aztec-network/eth-devnet && docker compose build && docker compose up)
+(cd ${REPO_ROOT}/spartan/aztec-network/eth-devnet && docker compose down -v && docker compose build && docker compose up)
```

Comment:
The `down -v` ensures volumes are cleaned up. Otherwise the containers will use the volumes with the config from an old `create.sh` run and fail to start.

Reply:
thanks for adding that